controllers/rcj_soccer_referee_supervisor/referee/supervisor.py

```diff
@@ -107,7 +108,10 @@ def __init__(
         self._update_positions()
 
         self.ball_reset_timer = 0
-        self.score_blue = self.score_yellow = 0
+        score_blue = os.environ.get("TEAM_B_INITIAL_SCORE", "0")
+        self.score_blue = int(score_blue) if score_blue != "" else 0
+        score_yellow = os.environ.get("TEAM_Y_INITIAL_SCORE", "0")
+        self.score_yellow = int(score_yellow) if score_yellow != "" else 0
```

Comment:
All of these env variables should be read in rcj_soccer_referee_supervisor.py and we pass their values to the referee during initialization.